_posts/2022-11-21-vws-rse-cop-summary.md

```diff
@@ -0,0 +1,130 @@
+together to create actionable suggestions for how to get members involved in a
+community of practice and how to make them feel invested and included. The authors
+hope that the results of this discussion can assist other RSEs in the
+creation of their own local CoP.
+
+## Starting a community of practice
+
+A CoP is a group of people who share some concern or passion for what they do
+and want to learn to do it better as they interact regularly. CoPs generally
+have 3 elements: (1) mutual engagement, (2) a joint enterprise, and (3) a
+shared repertoire. The three moderators recently took on the challenge of
+creating a formally-defined CoP at Sandia National Laboratories.
+
+At their inaugural meeting, they started by asking the question, “Who is the
+intended audience?” Because RSEng looks different across different organizations,
+to start a CoP off right it’s important to understand the intended audience.
+To determine this, they asked questions to the meeting attendees such as: What
+careers are representative within your organization? How can you cast a wide
+enough net to be inclusive without diluting the targeted community? Having an
+identity means that certain things are in scope and are part of that identity,
+and other things are not part of that identity. Poor scoping can stop a
+community before it even starts.
+
+> Tip: You don’t want to exclude potentially interested community members, so don’t start too small or refined in defining your intended audience, or you won’t get enough interest!
+
+Once the CoP has some initial interested participants, the next task is to
+define the goals of the CoP community. What is it trying to accomplish? What
+will it bring to and expect from the community members? For US-RSE, the mission
+has four parts: (1) community, (2) advocacy, (3) resources, and (4)
+diversity, equity, and inclusion. The founders of a new community can define
+its original mission – but should work with the community to refine it as time
+goes on. There should be a clear benefit for the community, whether that’s
+through access to information (e.g., providing resources on how to set up
+software infrastructure), a place for mutual commiseration, or a mechanism for
+finding collaborators.
+
+> Tip: It is okay for your goals to evolve as your community matures. Start with identifying common problems and promoting dialogue around those problems.
+
+## Community infrastructure
+
+Not everyone may have heard of the "Aesthetic-Usability Effect," but we have
+all experienced it. This effect refers to “users’ tendency to perceive
+attractive products as more usable. People tend to believe that things that
+look better will work better — even if they aren’t actually more effective or
+efficient.”[^1]
+
+[^1]: Moran, Kate. "The aesthetic-usability effect." Nielsen Norman Group: https://www.nngroup.com/articles/aesthetic-usability-effect (2017).
+
+Before beginning regular meetings and discussions, having a community
```

Comment:
Does this mean that what this paragraph describes makes the CoP more attractive? I think I would modify this sentence to something like: "Making regular meetings and discussions more "attractive" to capitalize on the "Aesthetic-Usability Effect," can, for instance, be achieved by having a community infrastructure where members can sign up for notifications, find information, resources, etc."
resources, etc.